docs/src/development/file-transfer.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Transfer files to and from your app"
+weight: 7
+sidebarTitle: Transfer files
+---
+
+After your app is built, its file system is read-only.
+This means that the only way you can edit your app's code is through Git.
+
+However, you can transfer files to and from your built app without using Git.
+To do so, you need to configure mounts or use an SSH client.
+
+[Mounts](../create-apps/app-reference.md#mounts) let you set up directories that remain writable after the build is complete.
+You can then transfer files directly to and from mounts inside your app
+with a single command via the [Platform.sh CLI](../administration/cli/_index.md).
+
+Alternatively, you can transfer files to and from your built app using an SSH client 
+such as `scp` or `rsync`.
```

Comment:
Reworked the introduction to make it clear from the get-go that 2 options are available to upload and download files to/from a built app:

- Mounts (cf. `mount:upload` and `mount:download` commands).
- SSH clients (now we have upload and download commands documented for both `scp` and `rsync`).